smart_aquarium/lib/ui/registerPage.dart

```diff
@@ -215,38 +265,15 @@ Future<int> registerUser(String username, String email, String password) async {
     if (response.statusCode == 200) {
       debugPrint('User is succesfully registered.');
       Map<String, dynamic> responseData = jsonDecode(response.body);
-      int userId = responseData['userId'];
-      return userId;
+
+      String token = responseData['token'];
+      return token;
     } else {
       debugPrint('Error during registration: ${response.statusCode}');
-      return -1;
+      return '';
     }
   } catch (e) {
     debugPrint('Error during registration: $e');
-    return -1;
-  }
-}
-
-Future<bool> checkIfUsernameExists(String username) async {
-  try {
-    var url = Uri.parse('$baseUrl/api/v1/users');
-    var response = await http.get(url);
-
-    if (response.statusCode == 200) {
-      debugPrint('Users data successfully fetched');
-      List<dynamic> users = jsonDecode(response.body);
-      for (dynamic user in users) {
-        if (user['userName'] == username) {
-          return true;
-        }
-      }
-      return false;
-    } else {
-      debugPrint('Error when fetching users: ${response.statusCode}');
-      return false;
-    }
-  } catch (e) {
-    debugPrint('Error when fetching users: $e');
-    return false;
+    return '';
```

Comment:
What would happen if insted of returning '' exception was thrown here

Reply:
I changed it so now when there's exception user will see an error on screen